sample.json

```diff
@@ -0,0 +1,15 @@
+{
+  "uuid" : "path/to/example.tif",
+  "projection" : "PROJCS["NAD83 / Massachusetts Mainland", ...",
+  "bbox" : [-180,-90,180,90],
+  "footprint" : "POLYGON((-180 -90, -180 90, 180 90, 180 -90, -180 -90))",
+  "gsd" : 0.35,
+  "file_size" : 1024,
```

Comment:
My bad, you did specify, apologies for reading your comment too quickly.

I guess in general I'm pro- keeping discoverable information out of metadata, since it creates a sync problem and things go stale. Another issue that I see with being inclusive of things like file size in the metadata is, there is probably a lot of information that we could include in the metadata, that we will probably not get right and get all of it in (or getting it all in would cause the metadata to blow up in size), that I can imagine the parsing step having to discover info for itself anyway.

That being said, I see your point about it being another step in parsing. If the consensus is that we are adding information to metadata (and taking on the potential sync burden) to ease the parsing step, then this piece of data (and probably many others) makes sense.